### PR TITLE
Create knapsack.cpp

### PR DESCRIPTION
**Pull request by indrajeet2000**
Hi, pls make my pull request hacktoberfest, hactoberfest-accepted, thank you for your consideration!

### CODE REVIEW

**Comment by indrajeet2000** [comment]
Please add my pull request and add hacktoberfest

**Comment by suryapratapsinghsuryavanshi** [comment]
> Please add my pull request and add hacktoberfest

I've made suggestion on the code part please rectify the, rest of the things are fine.

**Comment by indrajeet2000** [comment]
> > Please add my pull request and add hacktoberfest
> 
> I've made suggestion on the code part please rectify the, rest of the things are fine.

changes made successfully

File: src/CPP/CPP simple game [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+    for (int i = 0; i < width + 2; i++)
+        cout << "=";
+    cout << endl;
+    cout << "score:" << score << endl;
+
+}
+void Input()
+{
+    if (_kbhit())
```
Comment (suryapratapsinghsuryavanshi) [inline src/CPP/CPP simple game:70]: Fix this, I think this part is a platform dependent

**Review by suryapratapsinghsuryavanshi — CHANGES_REQUESTED** [review]

**Review by suryapratapsinghsuryavanshi — APPROVED** [review]

**Pull request merged**